index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
+    <title>Home</title>
+</head>
+
+<body>
+    <section class="header">
+        <div class="logo-Placeholder">
+            <p class="logo">My Logo</p>
+        </div>
+        <div class="menu"></div>
```

Comment:
- [ ] The menu icon is well placed, but I will suggest you place it inside a `nav` tag instead of a `div`. The `nav` tag gives a clear explanation that the menu handles navigations.

See guide: `<nav> <div class="menu"></div> </nav>`